components/NewPersonView/EventLink.tsx

```diff
@@ -17,14 +17,29 @@ const EventLink = ({ event }: Props): React.ReactElement => {
       mapFormIdToFormDefinition[event.formName]?.displayName || 'Form';
     const formTitle = event.title ? ` - ${event.title}` : '';
 
-    return (
-      <Link href={`/people/${event.personId}/submissions/${event.recordId}`}>
-        <a className={`lbh-link ${s.eventLink}`}>
-          {formName}
-          {formTitle}
-        </a>
-      </Link>
-    );
+    if (!event.deleted) {
+      return (
+        <Link href={`/people/${event.personId}/submissions/${event.recordId}`}>
+          <a className={`lbh-link ${s.eventLink}`}>
+            {formName}
+            {formTitle}
+          </a>
+        </Link>
+      );
+    } else {
+      return (
+        <>
+          <span>
+            {formName} {formTitle}{' '}
+          </span>
+          <Link
+            href={`/people/${event.personId}/submissions/${event.recordId}`}
+          >
+            <a>(deleted record)</a>
```

Comment:
Should this have the same class name as the existing code `className={`lbh-link ${s.eventLink}`}`?

Reply:
Good shout, I'll add the classname!